Resources/Prototypes/Entities/Mobs/Species/slime.yml

```diff
@@ -71,25 +71,25 @@
           sprite: Objects/Misc/handcuffs.rsi
           state: body-overlay-2
           visible: false
-        - map: [ "enum.Slots.IDCARD" ]
-        - map: [ "enum.Slots.GLOVES" ]
-        - map: [ "enum.Slots.SHOES" ]
-        - map: [ "enum.Slots.EARS" ]
-        - map: [ "enum.Slots.OUTERCLOTHING" ]
-        - map: [ "enum.Slots.EYES" ]
-        - map: [ "enum.Slots.BELT" ]
-        - map: [ "enum.Slots.NECK" ]
-        - map: [ "enum.Slots.BACKPACK" ]
```

Comment:
Given that this inherits from `MobHuman`, which in turn has `MobHumanBase`, does this actually need to be defined here at all, or can it just inherit.

Same goes for vox.yml I guess.

Reply:
true, althought i'll leave that up to the next pr